src/Core/Animation/Animation.hpp

```diff
@@ -8,36 +8,44 @@ namespace Ra {
 namespace Core {
 namespace Animation {
 
-class RA_CORE_API Animation
+/// An Animation is a set of key Poses given for certain time values.
```

Comment:
Can you add a quick comment explaining why this class is deprecated, and what are the recommended alternatives ?